README.md

```diff
@@ -6,4 +6,15 @@ In order to start an export:
 1. Copy `src/PlentyConfig.php.dist` to `src/PlentyConfig.php`.
 2. Set the correct values in `src/index.php`.
 3. Change directory to `src`.
-4. Run `php index.php`.
\ No newline at end of file
+4. Run `php index.php`.
+
+## Deployment & Release
+
+1. Ensure all necessary changes have been merged into the `develop` branch.
+1. Merge all changes from `develop` into `main`.
+1. Create a new release on GitHub. As branch use `develop`. This will **automatically** create
+a tag for this repository.
+1. Ensure that the **build passed successfully** and bump the version of the REST exporter in
+the importer by running `composer require findologic/plentymarkets-rest-export:^1.35`.
+   * Ensure that you specify `^` and **only** two versions e.g. `^1.35` or `^2.12`.
+1. From here on follow the usual importer deployment procedure.
```

Comment:
It would be convenient if you add a link to the deploy instructions at http://build.findologic.com/infrastructure/backend#deployment

Reply:
This is a public repository, why should we share internal links?

Reply:
Right, if it's public we should avoid it